src/js/components/Box/Box.js

```diff
@@ -131,7 +131,7 @@ class Box extends Component {
     );
 
     if (stateTheme) {
-      if (stateTheme.dark !== propsTheme.dark) {
+      if (stateTheme.dark !== propsTheme.dark && stateTheme.icon) {
```

Comment:
if we use a component using using `Grommet`,`stateTheme.icon` is empty. we should not invoke IconThemeContext